app/assets/javascripts/controllers/reconfigure/reconfigure_form_controller.js

```diff
@@ -198,6 +232,52 @@ ManageIQ.angular.app.controller('reconfigureFormController', ['$http', '$scope',
     vm.reconfigureModel.cb_bootable = false;
   };
 
+  vm.processAddSelectedNetwork = function() {
+    vm.reconfigureModel.vmNetworkAdapters.push(
+      {
+        name: 'to be determined',
```

Comment:
missing i18n? (and for mac)

Reply:
Good point. I will change it into `name: __('to be determined')`